pallets/messages/README.md

```diff
@@ -0,0 +1,65 @@
+# Messages Pallet
+
+Stores block-indexed data for a Schema using `OnChain` or `IPFS` payload location.
+
+## Summary
+
+Messages stores metadata and message payload data for a set Schema.
+Use of the Message pallet payloads is designed for time-centric data.
+Discovery is via the `MessagesInBlock` on new blocks or requesting messages from a block.
+Retrieval is via RPC.
+
+### Metadata vs Payload
+
+Messages have both metadata and payloads.
+The payload should always match the data structure or the messages is considered invalid.
```

Comment:
Should read "or the message is considered invalid" or "or the messages are considered invalid"

Reply:
```suggestion
The payload should always match the data structure or the message is considered invalid.
```